.env

```diff
@@ -33,7 +33,7 @@ DATABASE_URL=postgres://postgres:postgrespass@localhost:5432/postgres?schema=api
 KEYCLOAK_URL=https://keycloak.podkrepi.bg/auth
 KEYCLOAK_REALM=webapp-dev
 KEYCLOAK_CLIENT_ID=jwt-headless
-KEYCLOAK_SECRET=
+KEYCLOAK_SECRET=99043b92-c337-4869-8a3c-09822db0fd6c
```

Comment:
@Yakimov1337 That's not nice! :warning: 

Please review your staged changes for secrets before pushing them to the repo.

All secrets should be kept in `.env.local` files instead of `.env` files!